### PR TITLE
Snyk: Remove hardcoded secret

### PR DESCRIPTION
**Pull request by melaniemoody**
closes: CLAIMS-1070

### CODE REVIEW

File: jwt/jwt_test.go [hunk 1]
```diff
@@ -19,33 +20,8 @@ import (
...
-cPl/uSkZP42Als+9YMoFPU/xrrDPbUhu72EDrj3Bllnyb168jKLa4VBOccUvggxr
-Dm08I1hgYgdN5huzs7y6GeUCgYEAj+AZJSOJ6o1aXS6rfV3mMRve9bQ9yt8jcKXw
-5HhOCEmMtaSKfnOF1Ziih34Sxsb7O2428DiX0mV/YHtBnPsAJidL0SdLWIapBzeg
-KHArByIRkwE6IvJvwpGMdaex1PIGhx5i/3VZL9qiq/ElT05PhIb+UXgoWMabCp84
-OgxDK20CgYAeaFo8BdQ7FmVX2+EEejF+8xSge6WVLtkaon8bqcn6P0O8lLypoOhd
-mJAYH8WU+UAy9pecUnDZj14LAGNVmYcse8HFX71MoshnvCTFEPVo4rZxIAGwMpeJ
-5jgQ3slYLpqrGlcbLgUXBUgzEO684Wk/UV9DFPlHALVqCfXQ9dpJPg==
------END RSA PRIVATE KEY-----`)
+//go:embed dummyPrivateKey.txt
+var dummyPrivateKey []byte
```
Comment (rjj-work) [inline jwt/jwt_test.go:24]: Don't we still need to set the vaule somehow, i.e. read it in from a test file?
* move dummyPrivateKey.txt to testdata/dummyPrivateKey.txt
* use os.Readfile("testdata/dummyPrivateKey.txt")

ref: [os.ReadFile](https://pkg.go.dev/os#ReadFile)

File: jwt/jwt_test.go [hunk 2]
```diff
@@ -19,33 +20,8 @@ import (
...
-QGlW9rQ9mHEtUotIaRSJ6K/F3cxSGUEkX1a3FRnp6kPLcckC6NlqdNgNBd6rb2rA
-cPl/uSkZP42Als+9YMoFPU/xrrDPbUhu72EDrj3Bllnyb168jKLa4VBOccUvggxr
-Dm08I1hgYgdN5huzs7y6GeUCgYEAj+AZJSOJ6o1aXS6rfV3mMRve9bQ9yt8jcKXw
-5HhOCEmMtaSKfnOF1Ziih34Sxsb7O2428DiX0mV/YHtBnPsAJidL0SdLWIapBzeg
-KHArByIRkwE6IvJvwpGMdaex1PIGhx5i/3VZL9qiq/ElT05PhIb+UXgoWMabCp84
-OgxDK20CgYAeaFo8BdQ7FmVX2+EEejF+8xSge6WVLtkaon8bqcn6P0O8lLypoOhd
-mJAYH8WU+UAy9pecUnDZj14LAGNVmYcse8HFX71MoshnvCTFEPVo4rZxIAGwMpeJ
-5jgQ3slYLpqrGlcbLgUXBUgzEO684Wk/UV9DFPlHALVqCfXQ9dpJPg==
------END RSA PRIVATE KEY-----`)
+//go:embed dummyPrivateKey.txt
```
Comment (rjj-work) [inline jwt/jwt_test.go]: Consider create a `testdata` directory and use put the key file there.

**Review by rjj-work — COMMENTED** [review]
1. is there a way to run snyk on the modified code to confirm it is OK with the updated version?

I like the approach, just would like to put the test data in a `testdata` subdirectory.

File: jwt/jwt_test.go (hunk 2 above)
Comment (melaniemoody) [reply jwt/jwt_test.go]: added the testdata folder.

**Review by melaniemoody — COMMENTED** [review]
@rjj-work Thanks for your comments. I've downloaded Snyk extension for VS Code and ran it to verify Snyk approves the changes.

**Review by rjj-work — APPROVED** [review]

**Comment by melaniemoody** [comment]
Closing because this is not needed. We will add a Snyk file to ignore tests here

**Pull request closed without merging**